tests/test_tablib.py

```diff
@@ -306,6 +307,9 @@ def test_auto_format_detect(self):
         """Test auto format detection."""
         # html, jira, latex, rst are export only.
 
+        # Looks like openpyxl is indirectly causing a distutils deprecation warning
+        warnings.filterwarnings('ignore', module='distutils')
```

Comment:
Is this already reported upstream? If not let's do so, and perhaps include a link here so we can remove it one day.

Reply:
Are you able to upgrade to a newer virtualenv? If not, let's add a note and we can remove it when you upgrade.

Reply:
Just tested, and I get another warning from openpyxl when running pytest:

```
================================================= warnings summary ==================================================
/usr/local/lib/python3.7/site-packages/openpyxl/xml/functions.py:30
  /usr/local/lib/python3.7/site-packages/openpyxl/xml/functions.py:30: DeprecationWarning: defusedxml.lxml is no longer supported and will be removed in a future release.
    from defusedxml.lxml import fromstring as _fromstring, tostring
```

I have latest `openpyxl==3.0.0` and also latest `defusedxml==0.6.0`, but guess this doesn't really matter if openpyxl stops using defusedxml and uses another one instead.

Reply:
We're not relying on defusedxml.lxml, and don't install it. It's an optional dependency of openpyxl.

I've created an upstream PR to silence the warning:
* https://bitbucket.org/openpyxl/openpyxl/pull-requests/353/ignore-deprecation-warning-from/diff

We could also silence it, but let's see how this goes.